Gemfile

```diff
@@ -1,8 +1,3 @@
 source 'https://rubygems.org'
 
 gemspec
-
```

Comment:
Sorry didn't mean to cause any consternation.  I noticed that you put them in the gemspec and wrote up a comment.  Then I Fell asleep.  Failed to press "Comment" button.  Reviewing everything today.